auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -81,25 +81,27 @@ export default class NextPageMixin extends Vue {
         }
         return `/${nextStep}`
       case LoginSource.BCEID:
-        let bceditNextStep = '/'
+        let bceidNextStep = '/'
         // Redirect to TOS if no terms accepted
         // for invited users , handle user profile
         // Redirect to create team if no orgs
         // Redirect to dashboard otherwise
         if (!this.userProfile?.userTerms?.isTermsOfUseAccepted) {
-          bceditNextStep = Pages.USER_PROFILE_TERMS
+          bceidNextStep = Pages.USER_PROFILE_TERMS
+        } else if (!this.currentUser?.roles?.includes(Role.AccountHolder)) {
+          bceidNextStep = Pages.CHOOSE_AUTH_METHOD
         } else if (!this.currentOrganization && !this.currentMembership) {
```

Comment:
this and above check are same ? checking for membership vs accountholder role?

Reply:
I belive both are different cases, not 100% sure. will the account_holder role will be removed from backend if the user leaves from all his accounts?

Reply:
no bcsc as well, have the same check on that case also

Reply:
can you confirm with @sumesh-aot ?

Reply:
did u check this?

Reply:
If the user doesn't have any active accounts account_holder role won't be present.